pages/plugins/general_concepts.rst

```diff
@@ -6,6 +6,8 @@ API concepts
 Graylog uses certain patterns in its code bases to make it easier to write extensions.
 It is important to know about these to be successful in writing custom for it.
 
+You can browse the ava
```

Comment:
Forgot to save :-1: 
Fixed in https://github.com/Graylog2/documentation/pull/271/commits/5ed2fa2eec1caea545298357365ef5ac3be41e4d
![](http://25.media.tumblr.com/tumblr_lp2epd7QrL1qbhtrto1_500.gif)